app/views/search/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<iframe class="image-Deep-AI" src=<%= @image_options.deep_url %> width="520px" height="520px"></iframe>
```

Comment:
Images are displaying from the other websites via iframe, nothing is saved on our end at this point.

Reply:
I'm glad we got this to work!

Reply:
I think this is the best choice here given that we don't know which image the user will want to glitch yet.